File: scrape/weekly/week.py
# use new_week.py

import json
import requests
from bs4 import BeautifulSoup

execfile('../keys.py')

url = 'http://localhost/scrape/weekly/week.php'

data = { 'key': week_key } # make sure key matches in file

data_json = json.dumps(data)
payload = { 'data': data_json }
r = requests.post(url, data=payload)
print(r.text)

File: scrape/weekly/week.php
<?php

include_once('../keys.php');

if (isset($_POST['data'])) {
	$data = json_decode($_POST['data']);
	$key = $data->key;

	// make sure key matches in script
	if ($key === $week_key) {
		$path = $_SERVER['DOCUMENT_ROOT'];
		include_once($path . '/core/init.php');

		$freeChampions = json_decode(file_get_contents('https://na.api.pvp.net/api/lol/na/v1.2/champion?freeToPlay=true&api_key=' . $api_key))->champions;

		$tuesday = date('Y-m-d', strtotime('tuesday', strtotime('tomorrow')));

		foreach ($freeChampions as $champion) {
			$id = $champion->id;

			/*
			if ($insert->rotation_champion($id, $tuesday)) {
				echo "Inserted " . $id . " into database for date " . $tuesday . "\n";
			}
			 */
		}
	}
}

?>
